fetchconfig/gimx-fetchconfig.cpp

```diff
@@ -0,0 +1,118 @@
+﻿/*

```

Comment:
Please remove binary file above.

Reply:
Not sure it was clear enough: you commited the gimx-fetchconfig binary.
https://github.com/matlo/GIMX/blob/7b93f7ac0c0f0f7c89e962939944cee96504d080/fetchconfig/gimx-fetchconfig

Reply:
Yeah, I noticed that mistake. Don't worry, it will be gone in the next commit.